.github/workflows/generate-index.yml

```diff
@@ -69,16 +72,24 @@ jobs:
 
         # disable gpg pass prompt
         # https://stackoverflow.com/questions/49072403/suppress-the-passphrase-prompt-in-gpg-command
-      - name: sign the json
+      - name: sign the module firmware index json
         run: |
           gpg \
             --pinentry-mode=loopback \
             --passphrase "${{ secrets.PASSPHRASE }}" \
             --output boards/module_firmware_index.json.sig \
             --detach-sign boards/module_firmware_index.json
 
+      - name: sign the plugin firmware index json
+        run: |
+          gpg \
+            --pinentry-mode=loopback \
+            --passphrase "${{ secrets.PASSPHRASE }}" \
+            --output boards/plugin_firmware_index.json.sig \
+            --detach-sign boards/plugin_firmware_index.json
+
```

Comment:
Isn't it possible to use a single command line?

Reply:
I answered myself to that too :rofl: Since we are specifying `--output` we cannot have multiple `.sig` output files

Reply:
I think `--output` accepts only one argument, so I split the command line to avoid errors.